src/Clock.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-console */
+import React, { useEffect, useState } from 'react';
+
+type Props = {
+  clockName: string,
+};
+
+export const Clock: React.FC<Props> = ({ clockName }) => {
+  const [today, setToday] = useState(new Date());
```

Comment:
`today, setToday` 🤔 
  sounds a bit weird. Any ideas on how to improve its name?